docs/user/tutorial/index.rst

```diff
@@ -97,8 +97,8 @@ By now, you should have two email notifications:
 * Another one from Read the Docs, prompting you to "verify your email
   address". Click on the link to finalize the process.
 
-Finally, you created your account on Read the Docs
-and are ready to import your first project.
+Once done, your "Read the Docs" account is created
```

Comment:
```suggestion
Once done, your Read the Docs account is created
```

We haven't quoted Read the Docs like that anywhere else, but I understand the intention.